### PR TITLE
1 feature initial setup

### LINKED ISSUE #1: [FEATURE] Initial setup
Implement basic package functionality along with tests and docs.

### PR DESCRIPTION
**Pull request by MarcelRosier**
- basic inference functionality
- basic tests
- basic docs
- Readme setup
- Zenodo integration

### CODE REVIEW

File: stroke_segmentor/model_handler.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                probs = [
+                    inverter(x)["pred"] for x in decollate_batch(batch_data)
+                ]  # invert resampling if any
+                probs = torch.stack(probs, dim=0)
+                # print('inverted resampling', logits.shape)
+
+                all_probs.append(probs.cpu())
+
+            avg_probs = torch.mean(torch.stack(all_probs), dim=0)
+            labels = torch.argmax(avg_probs, dim=1).cpu().numpy().astype(np.int8)
```
[the remark this answers is not in the document]
Comment (MarcelRosier) [reply stroke_segmentor/model_handler.py]: labels are only 0/1

**Review by neuronflow — APPROVED** [review]
looks good on first glance, let's release so we can deploy and test :)

**Review by neuronflow — APPROVED** [review]
